ShoppingList/ShoppingList/ShoppingManager.swift

```diff
@@ -41,4 +45,20 @@ class ShoppingManager {
     func getShoppingItem(index: Int) -> ShoppingListItem {
         return shoppingList[index]
     }
+    
+    func highToLow() {
+        // filter -> 체크 / 미체크 분리
```

Comment:
불필요한 주석은 제거해주세요!
만약 주석을 남겨두고 싶다면 아래왁 같이 작성해주세요.
```suggestion
        // filter를 사용하여 체크 / 미체크 분리하여 정렬 후 합쳐서 리턴
        func highToLow() { ...
```

Reply:
@keenkim1202 완료!